src/R/lib_size_sc_V5_single_species.R

```diff
@@ -33,6 +33,10 @@ libtype <- args[10]
 hg_unfil <- read.csv(unfilt, header=T)
 hg_fil <- read.csv(filt, header=T)
 
+# ensure rows are in same order
+hg_unfil <- hg_unfil[order(hg_unfil$R1, hg_unfil$R2, hg_unfil$R3),]
```

Comment:
Add colnames so no $R1, $R2,$R3

Reply:
R1, R2, R3 are the colnames, I'm sorting the dataframes so that they're ordered by barcodes--what's wrong with this? Are you thinking of R's generic V1, V2, … column naming?